velox/tpch/gen/dbgen/build.cpp

```diff
@@ -85,22 +86,24 @@ static void gen_phone(DSS_HUGE ind, char* target, seed_t* seed) {
   RANDOM(exchg, 100, 999, seed);
   RANDOM(number, 1000, 9999, seed);
 
-  sprintf(target, "%02d", static_cast<int>(10 + (ind % NATIONS_MAX)));
-  sprintf(target + 3, "%03d", static_cast<int>(acode));
-  sprintf(target + 7, "%03d", static_cast<int>(exchg));
-  sprintf(target + 11, "%04d", static_cast<int>(number));
-  target[2] = target[6] = target[10] = '-';
+  std::string phone = fmt::format(
```

Comment:
I don't think introducing an extra allocation and a copy by using fmt is helpful here.
Why not combine these and check for the return value?
auto ret = snprintf(target, "%02d-%03d-%03d-%04d", .. ,);
// VELOX_CHECK_GT(ret, 0);

Reply:
Done.